ditatodocbook.sh

```diff
@@ -101,9 +101,16 @@ else
   echo "Did not find conversion.conf: Using default settings."
 fi
 
+OUTPUTDIR=$(echo "$OUTPUTDIR" | sed -r -e 's_^file:/+_/_' -e 's_/$__')
+
+outputdirabs="$basedir/$OUTPUTDIR"
+if [[ $(echo "$OUTPUTDIR"| grep "^(/") ]]; then
```

Comment:
That if clause looks weird and complex. So you want to check, if `$OUTPUTDIR` starts with a slash, right?

Why not using the following line?

```bash
if [[ $OUTPUTDIR = /* ]]; then
```

test:

```bash
x="/fooo/fooo"
[[ $x = /* ]] && echo "yes" || echo "no"
x="fooo/fooo"
[[ $x = /* ]] && echo "yes" || echo "no"
```

Output:

```
yes
no
```